main.js

```diff
@@ -1,10 +1,65 @@
+function addToLocalStorage(data) {
+  return localStorage.setItem('books', JSON.stringify(data));
+}
+
+function getFromLocalStorage(key) {
+  return JSON.parse(localStorage.getItem(key));
+}
+
+let collections = [];
+
+function addBookToCollection(title, author) {
+  const id = Math.random();
+  collections.push({ title, author, id });
+  addToLocalStorage(collections);
```

Comment:
# Optional
- Please use an array method that doesn't mutate the array: Kindly use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat) method instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push),

Reply:
Thanks a lot @OLIPLICHE :+1: ,
I made sure to implement your optional changes.